source/javascripts/services/_requestService.js.erb

```diff
@@ -542,6 +543,9 @@ angular.module("BitriseWorkflowEditor").service("requestService", function($q, $
 				if (dockerImage !== null) {
 					requestData.docker_image_name = dockerImage;
 				}
+				if (denTags !== null) {
```

Comment:
You can pass it without the `if` instead. In the case of `dockerImage` the null is a special value, meaning the default docker image. So it was needed to not be sent in that case, thus the if. There is no such thing for den tags, and also with the suggested change above, it will be assigend with undefined for cases when den tag setting is not an available feature, which is fine.